src/main/kotlin/interactor/GetCityHasTheHighestDifferentInApartmentRent.kt

```diff
@@ -0,0 +1,37 @@
+package interactor
+import model.CityEntity
+import utils.isNotNull
+import java.lang.IllegalStateException
+import java.util.NoSuchElementException
+
+class GetCityHasTheHighestDifferentInApartmentRent (
```

Comment:
Add Interactor at the top of the class name:
```suggestion
class GetCityHasTheHighestDifferentInApartmentRentInteractor (
```